deployment/cloudformation/majorkirby.py

```diff
@@ -0,0 +1,497 @@
+import sys
```

Comment:
Cool, I'll add a note to the README -- right now this file is a direct copy with no modifications in order to make it easy to update if there are changes.